redbot/cogs/mod/kickban.py

```diff
@@ -266,7 +266,7 @@ async def check_tempban_expirations(self):
                                 guild_tempbans.remove(uid)
             await asyncio.sleep(60)
 
-    @commands.command()
+    @commands.command(usage="<member> [reason]")
```

Comment:
Seems to me like renaming the variable to `member` would be better than having usages in most of the commands.